apps/appcontainer-node/packages/generic/src/appContainer.ts

```diff
@@ -223,13 +223,26 @@ export class AppContainer {
 				const app = this.apps.get(clientId)
 				if (app) app.lastPing = Date.now()
 			},
-			requestSpinDown: async (): Promise<void> => {
+			requestSpinDown: async (force?: boolean): Promise<void> => {
```

Comment:
I added a `force` parameter here, otherwise the workers requests to be spun down could often be ignored.